### DISCUSSION TITLE
Lazer cursor behaves strangely when using a graphics tablet

### DISCUSSION THREAD

**Discussion by 13Hz** [ppy/osu#26968] [Q&A] [answered]
Hello. I just switched from stable to lazer, updated to the latest current version. The first thing I saw was that my workspace does not match the one installed on the system. After checking the settings, I saw that in osu you can now set the area directly in the game. I disabled this option, but my cursor is stuck to the right edge of the screen, which I did not experience in stable. Perhaps I missed something in the settings?

OS: Windows 10
Tablet: Wacom CTL-672
[logs.zip](https://github.com/ppy/osu/files/14143727/logs.zip)

(I have two monitors, the second one is the main one)

https://github.com/ppy/osu/assets/39442071/e56893b7-748d-4edc-98ca-16be13e920bf

![image](https://github.com/ppy/osu/assets/39442071/c905bcc2-c611-43ef-89c0-f9102f32f1e2)

**Comment by 13Hz** [comment]
A small clarification to the above, before this I already launched lazer with exactly the same configuration on the old version (two or three months ago). This started happening after the last update

**Comment by Flamiii** [comment, marked as answer]
Do you have high precision mouse enabled in your lazer settings? If so, turn it off.

**Reply by 13Hz** [reply]
Surprisingly it worked, thanks for the help
